src/custom/UniversalFilter.tsx

```diff
@@ -104,6 +104,7 @@ function UniversalFilter({
                           defaultValue="All"
                           key={filterColumn}
                           value={selectedFilters[filterColumn]}
+                          variant="outlined"
```

Comment:
why?

Reply:
The Select element that is used here gives me an error saying that the type defined for it states that it needed a variant.
I added the line so that I could discuss the issue at the time of PR review.